mk/spksrc.spk.mk

```diff
@@ -12,6 +12,9 @@
 #  DSM_SCRIPT_FILES:  List of script files that are in the scripts folder within the spk file.
 #
 
+# Allow more extensive shell commands
+SHELL := /bin/bash
+
```

Comment:
why not in common.mk ? So it would be available in all *.mk files.

Reply:
Tested that and this makes much more sense.
Now part of #5007 and already merged.
thnx.